WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -137,9 +137,12 @@ final class AuthenticationManagerTests: XCTestCase {
         XCTAssertTrue(viewModel is NoSecureConnectionErrorViewModel)
     }
 
-    func test_it_presents_username_and_password_controller_for_non_jetpack_site() {
+    func test_it_presents_username_and_password_controller_for_non_jetpack_site_when_not_using_application_password_authentication() {
```

Comment:
Nit: should we also add a test case for when application password is used?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8769/commits/d224425421c86a5bb0c83c794592ef3b89aaf9c0